components/org.wso2.carbon.identity.tag.manager.core/src/main/java/org/wso2/carbon/identity/tag/manager/core/constant/TagMgtConstants.java

```diff
@@ -0,0 +1,209 @@
+                        "or 'or'."),
+        ERROR_CODE_EMPTY_FILTER_VALUE("60016", "Filter value cannot be empty",
+                "Filter value cannot be empty"),
+
+        // Server Errors.
+        ERROR_CODE_SORT_BY_NOT_IMPLEMENTED("65002", "Sort-By not implemented",
+                "Sort by option is not implemented"),
+        ERROR_CODE_SORT_ORDER_NOT_IMPLEMENTED("65003", "Sort-Order not implemented",
+                "Sort order is not implemented"),
+        ERROR_CODE_INVALID_DATA("65004", "Invalid data provided",
+                "Invalid data provided"),
+        ERROR_CODE_ERROR_GETTING_TAGS_FROM_TYPE_TENANT_UUID("65005", "Error loading tags",
+                "Error occurred while getting the attributes for the tags"),
+        ERROR_CODE_ERROR_LOADING_TAG("65006", "Error loading tags",
+                "Error occurred while getting the attributes for the tags"),
+        ERROR_CODE_ERROR_LOADING_TAG_TYPES("65007", "Error loading tag types",
+                "Error occurred while getting the list of tag types"),
+        ERROR_CODE_ERROR_GETTING_ASSOCIATIONS("65008", "Error getting tag-resource associations",
+                "Error occurred while all the tag-resource associations"),
+        ERROR_CODE_ERROR_GETTING_ASSOCIATION("65009", "Error getting association for tag",
+                "Error occurred while getting tag-resource association of the tag: %s , resource: %s"),
+        ERROR_CODE_ERROR_ADDING_TAG("65010", "Error while adding tag",
+                "Error occurred while adding the tag"),
+        ERROR_CODE_ERROR_ADDING_ASSOCIATION("65011", "Error while adding tag-resource association",
+                "Error occurred while adding tag-resource association"),
+        ERROR_CODE_ERROR_DELETING_ASSOCIATION("65012", "Error while deleting tag-resource association",
+                "Error occurred while deleting tag-resource association"),
+        ERROR_CODE_ERROR_DELETING_TAG("65013", "Error deleting tag",
+                "Error occurred while deleting the tag: %s"),
+        ERROR_CODE_ERROR_DELETING_TAGS("65014", "Error deleting tags",
+                "Error occurred while deleting the tags for the tenant: %s"),
+        ERROR_CODE_ERROR_UPDATING_TAG("65015", "Error while updating tag",
+                "Error occurred while updating tag: %s"),
+        ERROR_CODE_NO_UNIQUE_ID_FOR_TENANT("65016", "No uniqueId found for the tenant: %s",
+                "Error occurred while getting Tenant UUID."),
+        ERROR_CODE_ERROR_GETTING_TENANT_UUID("65017", "Error while getting the tenant UUID",
+                "Error occurred while getting the tenant UUID for tenant: %s"),
+        ERROR_CODE_ERROR_EMPTY_TAG_TYPES_LIST("65018", "Empty tag types list.",
+                "Tags have not been defined yet.");
+
+        private final String code;
+        private final String message;
+        private final String description;
+        public static final String ERROR_PREFIX = "TM-";
```

Comment:
Error code TM is used for tenant management as well https://github.com/wso2/identity-api-server/blob/636fdfc067d232acb7d2c94133ea7cac8ad76f5e/components/org.wso2.carbon.identity.api.server.tenant.management/org.wso2.carbon.identity.api.server.tenant.management.common/src/main/java/org/wso2/carbon/identity/api/server/tenant/management/common/TenantManagementConstants.java#L43
Shall we change this

Reply:
Shall we rename it to 'TMS'